packages/amplify-e2e-tests/src/__tests__/rds-model-v2.test.ts

```diff
@@ -125,10 +125,10 @@ describe('RDS Model Directive', () => {
       region,
     };
     const queries = [
-      'CREATE TABLE Contact (id VARCHAR(40) PRIMARY KEY, FirstName VARCHAR(20), LastName VARCHAR(50))',
-      'CREATE TABLE Person (personId INT PRIMARY KEY, FirstName VARCHAR(20), LastName VARCHAR(50))',
-      'CREATE TABLE Employee (ID INT PRIMARY KEY, FirstName VARCHAR(20), LastName VARCHAR(50))',
-      'CREATE TABLE Student (studentId INT NOT NULL, classId CHAR(1) NOT NULL, FirstName VARCHAR(20), LastName VARCHAR(50), PRIMARY KEY (studentId, classId))',
+      'CREATE TABLE Contact (id VARCHAR(40) PRIMARY KEY, firstName VARCHAR(20), lastName VARCHAR(50))',
```

Comment:
these formatting on existing setup is to ensure that these tests actually test what they're intending to w/o field mappings interfering. We have separate E2Es added for field mappings.